client/package.json

```diff
@@ -52,4 +53,4 @@
   },
   "proxy": "http://localhost:5000",
   "//": "https://ictrl.ca"
-}
+}
```

Comment:
Are JSON files required for future updates? If not, please update gitignore. 
Also needs a new line character

Reply:
package.json specifies the dependencies and running configurations of our project. I think they do need to be in the index.